.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,42 @@
+name: Integration Tests
+
+on: [pull_request, push]
+
+jobs:
+  fprime-integration:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        fprime-version: [[NASA-v1.5.3, 99cef07], [v2.0.0, 521516c] , [devel, 521516c]]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .
+      - name: clone fprime and gps-tutorial
+        run: |
+          git clone -b ${{ matrix.fprime-version[0] }} https://github.com/nasa/fprime.git
+          git clone https://github.com/fprime-community/gps-tutorial.git
+      - name: Checkout correct gps-tutorial
+        working-directory: gps-tutorial
+        run: git checkout ${{ matrix.fprime-version[1] }}
+      - name: Test generation and building on Ref
+        working-directory: fprime/Ref
+        run: |
+          fprime-util generate
+          fprime-util build
+      - name: Move fprime into gps-tutorial
```

Comment:
The GPS app should already be embedded a tagged version of fprime. I say we just checkout the GPS app and just use whatever fprime commit it includes in it's submodule.

Reply:
Got it!